case-studies/DLP.md

```diff
@@ -0,0 +1,60 @@
+---
+description: >-
+  My Best Life (MBL) is a programme run by New Philanthropy Capital (NPC). During Phase 2 of My Best Life, Founders and Coders assisted with the creation and facilitation of design sprint workshops which were completed by a team made up of representatives from NPC, Founders and Coders, charity partners UK Youth, The Mix and Leap Confronting Conflict, and members of UK Youth Voice. 
+---
+
+# Case Study: My Best Life
+
+## Introduction
+
+During MBL phase 1, NPC and the MBL charity partners had identified three problem areas faced by young people in reaching youth service provision. Phase 2 aimed to take each of these problem areas as a starting point for a design sprint - to find a digital solution idea for each and create a prototype that can be usability tested and potentially developed further.
```

Comment:
If you like, we can provide this link on "Phase 1"

https://www.thinknpc.org/blog/my-best-life/

Reply:
Also, I think it'd be good to mention that we tested the prototypes with Young People.